dask/dataframe/core.py

```diff
@@ -4951,14 +4951,19 @@ def quantile(df, q, method="default"):
 
     # currently, only Series has quantile method
     if isinstance(df, Index):
-        meta = pd.Series(df._meta_nonempty).quantile(q)
+        series_typ = type(df._meta.to_series())
```

Comment:
Can this just be `df._meta.to_series().quantile()`? I don't think we need the `series_type()` call, since IIUC that's doing `Series(series)`